### ISSUE TITLE
get location when not running?

### ISSUE THREAD

**Issue by porjo** [mapsme/omim#3713]
I'm running MAPS.ME 6.2.2-Google and have permissions set to ask me when attempting to get current location. I'm being prompted to allow the app to get location even when the app isn't running. Why is this necessary?

**Comment by rugk** [comment]
Do you have the saving of the route activated?

**Comment by rugk** [comment]
Do you have the saving of the route activated?

**Comment by porjo** [comment]
Do you mean `Settings -> Map -> Recent Track` ? That setting is currently off.

**Comment by rugk** [comment]
Yes, that's what I meant. (as I experienced it too: https://github.com/mapsme/omim/issues/1883)
